### ISSUE TITLE
How to tell if the user is not logged in when the app starts?

### ISSUE THREAD

**Issue by LiranBri** [Wizcorp/phonegap-facebook-plugin#339]
When the app starts, i need that If user already logged in --> Continue as logged in to the main page. If user is not logged in --> Present screen with "Push the Connect button to login"
now, when my app starts- i call the "FB.init" method. the problem is, that i don't have any callback to use so i know when it completes, and then check for the login status.

if i subscribe to the auth.statusChange event before the FB.init, then i only get this event raised when the user is already logged in. its not enough, because i also need to know when he is NOT logged in.

if i call "FB.getLoginStatus" just after the FB.init, it doesn't work since this all thing is async and it tries get the login status before the "fb.init()" completed.

in other words, i can't tell when the user is NOT LOGGED IN when the app starts.
